src/params/progression.jl

```diff
@@ -34,17 +38,38 @@ function sample_severity(rng::AbstractRNG, age::Real)
   elseif age < 80;  dist = severity_dists[5]
   else;             dist = severity_dists[6]
   end
+  idx = gender + 1 |> Int32
+  prob = hospitalization_probs[idx][age + 1]
+  if age < 0;       error("age should be non-negative")
+  elseif age < 40;  dist = Categorical([0,  1.0 - prob,  prob *(1-0.030927835051546393) ,  prob *0.030927835051546393])
+  elseif age < 50;  dist = Categorical([0,  1.0 - prob,  prob *(1-0.04161979752530934) ,  prob *0.04161979752530934])
+  elseif age < 60;  dist = Categorical([0,  1.0 - prob,  prob *(1-0.050980392156862744) ,  prob *0.050980392156862744])
+  elseif age < 70;  dist = Categorical([0,  1.0 - prob,  prob *(1-0.09044834307992203) ,  prob *0.09044834307992203])
+  elseif age < 80;  dist = Categorical([0,  1.0 - prob,  prob *(1-0.12330905306971904) ,  prob *0.12330905306971904])
+  else;             dist = Categorical([0,  1.0 - prob,  prob *(1-0.17335058214747737) ,  prob *0.17335058214747737])
+  end
   severity_int = rand(rng, dist)
   severity = rand(rng, dist) |> Severity
 end
 
-function sample_if_death(rng::AbstractRNG, severity::Severity)
+function sample_if_death(rng::AbstractRNG, severity::Severity, age::Real)
   severity_int = severity |> UInt8
   death_prob = death_probs[severity_int]
+  if severity != Critical
+    return false
+  elseif age < 0;       error("age should be non-negative")
+  elseif age < 40;  death_prob = 0.6666666666666666
```

Comment:
same here. Ideally they should be read from some external file, but for now they can be at least stored in some const-array